IFC4x3/PropertyConstants/n/NOTHING_09aBFduaLDvRm2j7p$eK9N.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" Name="NOTHING" UniqueId="0990b3e7-e245-4de5-bc02-b47cffa14257" />
```

Comment:
Have to delete.

Reply:
Done. Commit : e5e747f8ce6e67ed1e5301b5fb1fba218df02d67